include/arch/arm/cortex_m/mpu/arm_mpu_v8m.h

```diff
@@ -77,17 +88,26 @@
 	 | \
 	 (NORMAL_I_WB_NT | R_ALLOC_W_ALLOC))
 
+#define NORMAL_OUTER_INNER_NON_CACHEABLE \
+	((NORMAL_O_NON_C | (R_NON_W_NON << 4)) \
+	 | \
+	 (NORMAL_I_NON_C | R_NON_W_NON))
+
```

Comment:
Could we enable R&W allocate on wirite-back cache?

Reply:
I think we do. These macros are used for non-cacheable attributions.